public/pages/createDetector/containers/CreateDetector.tsx

```diff
@@ -119,9 +119,17 @@ export function CreateDetector(props: CreateADProps) {
         `/detectors/${detectorResp.data.response.id}/configurations/`
       );
     } catch (err) {
-      toastNotifications.addDanger(
-        getErrorMessage(err, 'There was a problem creating detector')
+      const resp = await dispatch(
```

Comment:
How about we use _count API?

Reply:
I'm not sure how that would apply here; I'm just leveraging our existing `searchDetector` function. Similar statement used [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/c5bb7c45b707a8d944e2fb9b252c95f77e468c38/public/pages/createDetector/containers/CreateDetector.tsx#L172) and I wanted to keep it consistent.

Reply:
Wait I see; query .opendistro-anomaly-detectors directly. Let me try

Reply:
Maybe another easy way is passing detector count to detector creation page. As dashboard and detector list page has already got total detector count.

Reply:
Oh, sorry, if pass detector count, we can't avoid the case that user open creation page directly by typing url. So we still need to count the total detector on creation page.

Reply:
I'm concerned it's unnecessary to add a separate function to get the count using the _count API; this would add a lot of code for a function that may not need to be used outside of this case.

Reply:
Got it. Let's use this now as the performance is acceptable considering we have 1k detectors at most. Let's tune it later.

Reply:
As discussed offline, will leave as-is for now. Current fix doesn't affect performance regarding showing the callout. Future improvements to this and other parts of code may use _count API.